packages/meta/README.md

```diff
@@ -49,7 +49,7 @@ Notably these features are missing:
 > NOTE: The API is considered unstable and may change at any time.
 > While in alpha we are gathering feedback from developers about the usability and ergonomics of the API.
 > To provide feedback please [open an issue](https://github.com/temporalio/sdk-node/issues) and feel free to
-> talk to us on the [community slack channel](https://slack.com/app_redirect?app=TNWA8QCGZ&channel=nodejs-sdk).
+> post on our [community discourse](https://community.temporal.io/).
```

Comment:
```suggestion
> talk to us on the [community slack channel](https://slack.com/app_redirect?app=TNWA8QCGZ&channel=nodejs-sdk) or post on our [community discourse](https://community.temporal.io/).
```

Reply:
I'm not sure the slack channel is open to the community.  When I click the link I see:

![image](https://user-images.githubusercontent.com/665024/129930900-1a4179d0-66e5-4b1d-9e4c-a4e35627851d.png)